test/Microsoft.ML.Benchmarks/Microsoft.ML.Benchmarks.csproj

```diff
@@ -1,4 +1,5 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\ExternalBenchmarkDataFiles.props" /> 
```

Comment:
You can use `$(RepoRoot)\build\....` instead. That way the project can live anywhere in the filesystem.